### ISSUE TITLE
[Frontend V2] Set font-weight 300 and reduce width of row separator on My submissions and All submissions tab

### ISSUE THREAD

**Issue by Ram81** [Cloud-CV/EvalAI#3338]
### Description

Font-weight is inconsistent on the `My submissions` and `All submissions` page. We should set font-weight: 300 for both tabs and reduce the width of row separator. Refer to frontend v1 css for the style.

**Comment by gautamjajoo** [comment]
Can I work on it?

**Comment by Ram81** [comment]
@gautamjajoo  sure go ahead.

**Comment by gautamjajoo** [comment]
There is no zoom on hovering a row instead the text gets bold. Should that be removed?

**Comment by Ram81** [comment]
Yes, the highlighting of the text should be removed. I'll update the description

**Comment by gautamjajoo** [comment]
@Ram81 Would this look better for the row separator? Or we should have a line above the expanded row as well?

![Screenshot from 2021-03-24 20-51-27](https://user-images.githubusercontent.com/24366008/112336230-e9539a80-8ce2-11eb-9018-21adc6045f8f.png)
![Screenshot from 2021-03-24 20-51-42](https://user-images.githubusercontent.com/24366008/112336240-ea84c780-8ce2-11eb-8596-cb77672e5d4f.png)
